main.rb

```diff
@@ -1,23 +1,23 @@
 require_relative './nameable'
+require_relative './association'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age, :parent_permission
```

Comment:
Getters and setters for rentals (:rentals) need to be set in order to read and change the values of the rentals, so kindly add it to the attr_accessor instead of the attr_reader :+1: